2-collectors/scala-stream-collector/src/main/scala/com.snowplowanalytics.snowplow.collectors.scalastream/CollectorRoute.scala

```diff
@@ -14,14 +14,17 @@
  */
 package com.snowplowanalytics.snowplow.collectors.scalastream
 
-import akka.http.scaladsl.model.{ContentType, HttpResponse}
+import akka.http.scaladsl.model.{ContentType, HttpResponse, StatusCode, StatusCodes}
 import akka.http.scaladsl.model.headers.HttpCookiePair
 import akka.http.scaladsl.server.{Directive1, Route}
 import akka.http.scaladsl.server.Directives._
+import com.snowplowanalytics.snowplow.collectors.scalastream.monitoring.BeanRegistry
```

Comment:
you can `import monitoring.BeanRegistry`

also we try to separate internal from external imports by a newline, e.g. https://github.com/jspc/snowplow/blob/30d01e8909ffb76239759bdedb70ee473cecd4d0/2-collectors/scala-stream-collector/src/main/scala/com.snowplowanalytics.snowplow.collectors.scalastream/Collector.scala#L27-L29

Reply:
I was in two minds about the relative import, but I'm happy to follow your lead.